network.go

```diff
@@ -47,6 +47,9 @@ type Network interface {
 
 	// EndpointByID returns the Endpoint which has the passed id. If not found, the error ErrNoSuchEndpoint is returned.
 	EndpointByID(id string) (Endpoint, error)
+
+	// EndpointByContainerID returns the Endpoint backed by a container on a network.
+	EndpointByContainerID(containerID string) (Endpoint, error)
```

Comment:
I think we should not couple the container id to Endpoint.
I'd rather provide a method to the `Sandbox` interface which returns the `Endpoint`(s) the sandbox is connected to.